src/routes/AgreementViewRoute.js

```diff
@@ -270,11 +270,32 @@ class AgreementViewRoute extends React.Component {
     history.push(`${urls.agreementEdit(match.params.id)}${location.search}`);
   }
 
+  handleExportAgreement = () => {
+    const { resources, stripes: { okapi } } = this.props;
+    const { id, name } = get(resources, 'agreement.records[0]', {});
+
+    return fetch(`${okapi.url}/erm/sas/${id}/export/current`, {
+      headers: {
+        'X-Okapi-Tenant': okapi.tenant,
+        'X-Okapi-Token': okapi.token,
+      },
+    }).then(response => response.blob())
+      .then(blob => {
+        const url = window.URL.createObjectURL(blob);
```

Comment:
It might be worth writing a `download` method now that we do this so often so we don't repeat ourselves: 
```
downloadBlob = blob => {
  const url = window.URL.createObjectURL(blob);
  const a = document.createElement('a');
  a.href = url;
  a.download = name;
  document.body.appendChild(a);
  a.click();
  a.remove();
}
```

And then call it from all these various `handleExport***` methods.